sources/packages/backend/apps/api/src/services/student-appeal/student-appeal.service.ts

```diff
@@ -59,7 +59,7 @@ export class StudentAppealService extends RecordDataModelService<StudentAppeal>
   async saveStudentAppeals(
     applicationId: number,
     userId: number,
-    studentAppealRequests: StudentAppealRequestModel[],
+    studentAppealRequests: StudentAppealRequestAPIInDTO[],
```

Comment:
The idea here was to avoid the service relying on a DTO class. Is there a reason to have it changed?

Reply:
My idea was to remove duplicate code as we have `StudentAppealRequestModel` and `StudentAppealRequestAPIInDTO`. As I reverted student appeal changes, I'm reverting this one too. Thanks!

Reply:
Thanks for reverting it. The code should not be considered duplicated in this case. The DTO and the service model belong to different layers and they can evolve differently.
Now that we have the API service layer more dedicated to the API it is not as a big deal as it used to be but ideally, we try to avoid using API DTOs at the service level.
@dheepak-aot @guru-aot please correct me if I am not saying something accurate here.